app/js/arethusa.core/services/id_handler.js

```diff
@@ -53,9 +71,12 @@ angular.module('arethusa.core').service('idHandler', [
         res = sentenceId ? res.replace('%s', sentenceId) : res.replace('%s-', '');
       }
       return res;
+      
+      function parseId(id) {
```

Comment:
True. I thought about just doing away with the function altogether, but it's still good to have the code reuse for simpler adjusting to changes in the ID format.